java/refactoring.java/src/org/netbeans/modules/refactoring/java/ui/CopyFullyQualifiedClassNameAction.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.netbeans.modules.refactoring.java.ui;
```

Comment:
The class does not use any `java.refactoring` concepts; shouldn't it be rather in some `org.netbeans.modules.java.sourceui` or even lower (i.e. next to JavaDataObject) ?

Reply:
Actually I struggled with the location. I already thought about putting it next to `JavaDataObject`, but it seems to be misplaced for some kind of a ui component.

Going to move to `org.netbeans.modules.java.ui` which seems to be a better place than now.